haskell-tests/Genesis/ParametersSpec.hs

```diff
@@ -0,0 +1,44 @@
+{-# LANGUAGE DataKinds #-}
+
+module Genesis.ParametersSpec (tests) where
```

Comment:
Provide a summary of what this is testing. Looks like just JSON serialization, but the module name implies a more general test.